beacon-chain/p2p/testing/p2p.go

```diff
@@ -48,11 +48,6 @@ type TestP2P struct {
 	LocalMetadata   metadata.Metadata
 }
 
-func (p *TestP2P) BroadcastBlob(ctx context.Context, subnet uint64, blobSidecar *ethpb.SignedBlobSidecar) error {
-	//TODO implement me
-	panic("implement me")
-}
-
```

Comment:
Why did you delete these? These methods were missing and causing the test types to not fulfill the interface.

Reply:
I deleted the duplications. I think they got added twice by two different PRs

Reply:
gotcha ty